lib/common/api/deprecate.js

```diff
@@ -46,6 +46,26 @@ deprecate.log = (message) => {
   }
 }
 
+// Deprecate an event.
+deprecate.event = (emitter, oldName, newName) => {
+  let warned = false
+  return emitter.on(newName, function (...args) {
+    // There are no listeners for this event
+    if (this.listenerCount(oldName) === 0) { return }
+    // noDeprecation set or if user has already been warned
+    if (warned || process.noDeprecation) { return }
+    warned = true
+    // Event name starts with a -, which means it's an internal event.
+    // For deprecation, that means the event cannot be used anymore.
+    if (!newName.startsWith('-')) {
```

Comment:
The comment says "Event name starts with a -", but the code says that it doesn't: `if (!newName.startsWith('-'))`.
Who's lying? )

Reply:
No one or both. 😄
I will update.